pkg/virt-controller/watch/pool_test.go

```diff
@@ -180,18 +175,19 @@ var _ = Describe("Pool", func() {
 			client = kubevirtfake.NewSimpleClientset()
 
 			// Set up mock client
-			virtClient.EXPECT().VirtualMachineInstance(metav1.NamespaceDefault).Return(vmiInterface).AnyTimes()
-			virtClient.EXPECT().VirtualMachine(metav1.NamespaceDefault).Return(vmInterface).AnyTimes()
+			virtClient.EXPECT().VirtualMachineInstance(metav1.NamespaceDefault).Return(client.KubevirtV1().VirtualMachineInstances(metav1.NamespaceDefault)).AnyTimes()
+			virtClient.EXPECT().VirtualMachine(metav1.NamespaceDefault).Return(client.KubevirtV1().VirtualMachines(metav1.NamespaceDefault)).AnyTimes()
 
 			virtClient.EXPECT().VirtualMachinePool(testNamespace).Return(client.PoolV1alpha1().VirtualMachinePools(testNamespace)).AnyTimes()
 
-			client.Fake.PrependReactor("*", "*", func(action testing.Action) (handled bool, obj runtime.Object, err error) {
+			client.Fake.PrependReactor("*", "*", func(action k8stesting.Action) (handled bool, obj runtime.Object, err error) {
+				defer GinkgoRecover()
```

Comment:
Why only in this one?

Reply:
It is a leftover actually. I mean, that is correct and we should do it for every PrependReactor and I'd prefer to address it with a separate PR for several files. I am going to drop it, or do you prefer to start doing it?

Reply:
Your call, just thought I'd point out the inconsistency!

Reply:
Don't we need it only in case the code is executed in different goroutine? This should not be a case most of the time.